sw/device/tests/pattgen_ios_test.c

```diff
@@ -27,26 +30,50 @@
    and output will be collected at the PADS and compared in SV test bench.
  */
 
-OTTF_DEFINE_TEST_CONFIG();
+OTTF_DEFINE_TEST_CONFIG(.enable_uart_flow_control = true);
+
+enum {
+  kTestCmdTimeoutUsec = 5000000,
+};
+
+/* Macro to define and use variable that need to be stored in flash
+ * for DV tests and RAM for real tests, to enable backdoor access. */
+#define DEFINE_BACKDOOR_VAR(type, name, default_val)                       \
```

Comment:
Should this be moved to macros.h?

Reply:
I am not sure I want to make such a change in this PR. We have many instances spread over several tests, I think it's better to create a PR just for that.

Reply:
yes that would be a good option. I'll look into this and create another PR.